volatility3/framework/symbols/windows/__init__.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, *args, **kwargs) -> None:
         self.set_type_class("_VACB", extensions.VACB)
         self.set_type_class("_POOL_TRACKER_BIG_PAGES", pool.POOL_TRACKER_BIG_PAGES)
         self.set_type_class("_IMAGE_DOS_HEADER", pe.IMAGE_DOS_HEADER)
+        self.set_type_class("_KTIMER", extensions.KTIMER)
```

Comment:
Will this always be present, even in old versions of windows?  If this symbol isn't present in some old versions then this should be `optional_set_type_class`.  Otherwise this is fine, just want to check before we merge...

Reply:
Yes, it should be in every version. And I've run this against several versions all the way back to XP and it works.

Reply:
Cool, happy with that then, just wanted to check  5:)